rediscluster/client.py

```diff
@@ -1104,7 +1140,7 @@ def _random_good_hashslot_key(self, hashslot):
         Generate a good random key with a low probability of collision between any other key.
         """
         # TODO: Check if the key exists or not. continue to randomize until a empty key is found
-        random_id = "{{0}}{1}".format(hashslot, self._random_id())
+        random_id = "{{{0}}}{1}".format(hashslot, self._random_id())
```

Comment:
What problem do this solve? Will this not break the output from this method when x2 { is kept in the output string?